crates/code_generation/src/run.rs

```diff
@@ -46,6 +46,10 @@ pub fn generate_run_function(cyclers: &Cyclers) -> TokenStream {
 
             #start_cyclers
 
+            #[cfg(feature = "systemd")]
+            systemd::daemon::notify(false, [(systemd::daemon::STATE_READY, "1")].iter())
```

Comment:
Use [`std::iter::once()`](https://doc.rust-lang.org/std/iter/fn.once.html) here.

Reply:
Done, changed it also from my copy source in hula ;)